libs/commons/Fpath_.ml

```diff
@@ -37,3 +37,8 @@ module Operators = struct
 end
 
 let readable ~root path = Filename_.readable ~root:!!root !!path |> Fpath.v
+
+(* Make the fpath "forget" whether the last element is a directory or not. *)
+let to_string_no_trailing_slash x =
+  let dir, file = Fpath.split_base x in
+  Fpath.(append dir file) |> Fpath.to_string
```

Comment:
What's going on here? This seems unrelated to both the PR summary and the other code in the PR.